packages/awesome-git/awesome-git.pacscript

```diff
@@ -11,8 +11,8 @@ pkgname="awesome"
 pkgdir="/usr/src/pacstall/${name}"
 description="awesome window manager"
 url="https://github.com/awesomeWM/${pkgname}.git"
-depends="libdbus-1-dev libxcb-xfixes0-dev libxcb-randr0-dev libxcb-xinerama0-dev libxcb-icccm4-dev libxcb-xtest0-dev liblua5.3-dev libcairo2-dev dbus libgdk-pixbuf2.0-dev libxdg-basedir-dev libxkbcommon-x11-dev lua5.3 lua-lgi-dev libpango1.0-dev libstartup-notification0-dev libxcb-cursor-dev libxcb-keysyms1-dev libxcb-util-dev libxcb-xrm-dev libxfixes-dev cmake"
-build_depends="asciidoc docbook-xsl git imagemagick lua-ldoc xmlto"
+depends="libdbus-1-dev libxcb-xfixes0-dev libxcb-randr0-dev libxcb-xinerama0-dev libxcb-icccm4-dev libxcb-xtest0-dev liblua5.3-dev libcairo2-dev dbus libgdk-pixbuf2.0-dev libxdg-basedir-dev libxkbcommon-x11-dev lua5.3 lua-lgi-dev libpango1.0-dev libstartup-notification0-dev libxcb-cursor-dev libxcb-keysyms1-dev libxcb-util-dev libxcb-xrm-dev libxfixes-dev"
+build_depends="asciidoc cmake docbook-xsl git imagemagick lua-ldoc xmlto"
```

Comment:
Most of the `depends` are actually `build_depends`.

Reply:
Suggest the change.